docs/user_management.md

```diff
@@ -0,0 +1,37 @@
+# How to give users the permissions to set up a run 
+Users that use the NPG pipeline could ask, through a ticket, to be 
+granted some permissions to set up runs. Permissions should be granted 
+only to those that are well known to the group. So, before adding users 
+to a permission group, it is better to ask the team (if that person is 
+unknown to you).
```

Comment:
As discussed, probably remove the last two sentences and move them to the internal training manual